scripts/agent.yml

```diff
@@ -195,18 +195,12 @@ tasks:
           aws_key: ${aws_key}
           aws_secret: ${aws_secret}
           s3_copy_files:
-            - {
-                "source":
-                  {
-                    "path": "evergreen/smoke/${build_id}-${build_variant}/evergreen-${task_name}-${revision}",
-                    "bucket": "mciuploads",
-                  },
-                "destination":
-                  {
-                    "path": "evergreen/smoke/${build_id}-${build_variant}/evergreen-${task_name}-${revision}-copy",
-                    "bucket": "mciuploads",
-                  },
-              }
+            - source:
+              path: "evergreen/smoke/${build_id}-${build_variant}/evergreen-${task_name}-${revision}"
+              bucket: mciuploads
+            - destination:
```

Comment:
`source` and `destination` have to be under the same YAML list item since `s3_copy_files` is a slice of source/destination parameters.

Reply:
done